airbyte-webapp/src/hooks/services/Feature/FeatureService.test.tsx

```diff
@@ -1,12 +1,13 @@
 import { render } from "@testing-library/react";
 import { renderHook } from "@testing-library/react-hooks";
+import React from "react";
 import { useEffect } from "react";
 
 import { FeatureService, IfFeatureEnabled, useFeature, useFeatureService } from "./FeatureService";
 import { FeatureItem, FeatureSet } from "./types";
 
 const wrapper: React.FC<React.PropsWithChildren<unknown>> = ({ children }) => (
-  <FeatureService features={[FeatureItem.AllowCreateConnection, FeatureItem.AllowSync]}>{children}</FeatureService>
```

Comment:
Could you replace the removed `AllowCreateConnection` via another feature here please. Currently some of those tests in this file are simply not testing anything reasonable anymore, since the feature is gone. E.g. the  "Workspace features can disable default features", does not try to disable any default feature anymore and so the test isn't testing anymore what it should. We should make sure that even with this flag removed all tests in this file are still testing what they are meant to test.

Reply:
Oh, sure! Will fix that!